### PR TITLE
dependencies: upgrade to angular 14

### PR DESCRIPTION
**Pull request by jma**
## Why are you opening this PR?

- Which task/US does it implement?
- Which issue does it fix?

## How to test?

- What command should I have to run to test your PR?
- What should I test through the UI?

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Extracted translations?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,80 +8,86 @@
...
+    "@angular/forms": "^14.2.5",
+    "@angular/platform-browser": "^14.2.5",
+    "@angular/platform-browser-dynamic": "^14.2.5",
+    "@angular/router": "^14.2.5",
     "@biesbjerg/ngx-translate-extract": "^7.0.3",
     "@biesbjerg/ngx-translate-extract-marker": "^1.0.0",
     "@ngx-formly/bootstrap": "^5.12.4",
     "@ngx-formly/core": "^5.12.4",
     "@ngx-translate/core": "^13.0.0",
     "@ngx-translate/http-loader": "^6.0.0",
```
Comment (Garfield-fr) [inline package.json:31]: `"@ngx-translate/http-loader": "^7.0.0"` ?

File: package.json [hunk 2]
```diff
@@ -8,80 +8,86 @@
...
+    "ngx-bootstrap": "^9.0.0",
+    "ngx-spinner": "^14.0.0",
+    "ngx-toastr": "^15.2.0",
     "npm-registry-fetch": "^9.0.0",
     "path-parse": ">=1.0.7",
     "postcss": ">=8.2.10",
     "rxjs": "^6.6.7",
     "serialize-javascript": "^5.0.1",
-    "simplemde": "^1.11.2",
     "socket.io": "^3.1.2",
```
Comment (Garfield-fr) [inline package.json:56]: "socket.io": "^4.5" ?

File: package.json [hunk 3]
```diff
@@ -8,80 +8,86 @@
...
+    "@angular-eslint/builder": "14.1.2",
+    "@angular-eslint/eslint-plugin": "14.1.2",
+    "@angular-eslint/eslint-plugin-template": "14.1.2",
+    "@angular-eslint/schematics": "14.1.2",
+    "@angular-eslint/template-parser": "14.1.2",
+    "@angular/cli": "^14.2.5",
+    "@angular/compiler-cli": "^14.2.5",
+    "@angular/language-service": "^14.2.5",
     "@ngx-formly/schematics": "^5.12.4",
     "@types/jasmine": "^3.6.9",
```
Comment (Garfield-fr) [inline package.json:75]: "@types/jasmine": "^4.3" ?

File: package.json [hunk 4]
```diff
@@ -8,80 +8,86 @@
...
-    "@types/marked": "^2.0.3",
     "@types/node": "^12.20.7",
```
Comment (Garfield-fr) [inline package.json:76]: `"@types/node": "^18.8"` ?

File: package.json [hunk 5]
```diff
@@ -8,80 +8,86 @@
...
-    "@types/simplemde": "^1.11.7",
-    "codelyzer": "^6.0.0",
+    "@typescript-eslint/eslint-plugin": "5.37.0",
+    "@typescript-eslint/parser": "5.37.0",
+    "eslint": "^8.23.1",
     "husky": "^4.3.8",
     "jasmine-core": "~3.6.0",
```
Comment (Garfield-fr) [inline package.json:81]: `"jasmine-core": "~4.3.0"` ?

File: package.json [hunk 6]
```diff
@@ -8,80 +8,86 @@
...
     "jasmine-spec-reporter": "~5.0.0",
```
Comment (Garfield-fr) [inline package.json:82]: `"jasmine-spec-reporter": "~7.0.0"`

File: package.json [hunk 7]
```diff
@@ -8,80 +8,86 @@
...
     "karma": "^6.3.4",
```
Comment (Garfield-fr) [inline package.json:83]: `"karma": "^6.4"` ?

File: package.json [hunk 8]
```diff
@@ -8,80 +8,86 @@
...
     "karma-chrome-launcher": "~3.1.0",
     "karma-jasmine": "~4.0.0",
```
Comment (Garfield-fr) [inline package.json:85]: `"karma-jasmine": "~5.1.0"` ?

File: package.json [hunk 9]
```diff
@@ -8,80 +8,86 @@
...
     "karma-jasmine-html-reporter": "^1.5.4",
```
Comment (Garfield-fr) [inline package.json:86]: "karma-jasmine-html-reporter": "^2.0"

File: package.json [hunk 10]
```diff
@@ -8,80 +8,86 @@
...
-    "ng-packagr": "^11.2.4",
+    "ng-packagr": "^14.2.1",
     "protractor": "^7.0.0",
     "ts-node": "^9.1.1",
     "tslint": "~6.1.0",
```
Comment (Garfield-fr) [inline package.json:90]: remove ?

**Review by Garfield-fr — APPROVED** [review]

**Review by zannkukai — APPROVED** [review]

File: package.json (hunk 6 above)
Comment (jma) [reply package.json:63]: Unfortunately not we need to fix some deprecations in the test files first.

**Pull request merged**